style.css

```diff
@@ -0,0 +1,295 @@
+/**
+ *
```

Comment:
This is the wrong CSS file I'm afraid - the one in syles is the one you want this is why the styles aren't showing up. I think the work you have done is very good though, it's very close
https://github.com/CodeYourFuture/bikes-for-refugees/blob/main/styles/style.css